sessionctx/variable/noop.go

```diff
@@ -110,7 +110,9 @@ var noopSysVars = []*SysVar{
 	{Scope: ScopeGlobal, Name: InnodbCmpPerIndexEnabled, Value: Off, Type: TypeBool, AutoConvertNegativeBool: true},
 	{Scope: ScopeGlobal, Name: "innodb_ft_server_stopword_table", Value: ""},
 	{Scope: ScopeNone, Name: "performance_schema_max_file_instances", Value: "7693"},
-	{Scope: ScopeNone, Name: "log_output", Value: "FILE"},
+	{Scope: ScopeGlobal, Name: "log_output", Value: "FILE", Validation: func(vars *SessionVars, normalizedValue string, originalValue string, scope ScopeFlag) (string, error) {
```

Comment:
I think this should be moved to other variables, it is not no-op anymore.

Reply:
While it is straight forward to accomodate, I don't quite agree. 
It is still a no-op as any attempt to set the parameter will result to clear error messages. 
Isn't this the same as TxReadOnly, SecureAuth and a few other parameters in noop.go ?

Reply:
Yes, I agree with @lucifinil - it is still a `noop`, even though it now does do some validation.

We use the `noop` list for some purposes, like auto-generating documentation. If we move it to the other list it will appear as "missing from docs", but that wouldn't be correct.